plugin/trino-hive/src/main/java/io/trino/plugin/hive/TableAlreadyExistsException.java

```diff
@@ -34,6 +34,11 @@ public TableAlreadyExistsException(SchemaTableName tableName, String message)
         this(tableName, message, null);
     }
 
+    public TableAlreadyExistsException(SchemaTableName tableName, Throwable cause)
```

Comment:
You could make use of this constructor in 

https://github.com/trinodb/trino/blob/794ca2a8ce40d847054cbcec8cba2bb7789ecaa1/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java#L566